src/lib.rs

```diff
@@ -25,11 +25,9 @@ use std::io::prelude::*;
 use std::io::{self, Error, ErrorKind, SeekFrom};
 use std::iter::repeat;
 use std::mem;
+use std::path::{Path, PathBuf};
 use std::str;
 
-#[cfg(feature = "nightly")]
```

Comment:
In my latest patch, I added an `if cfg!(feature = "nightly")` guard to the not(unix) version of set_perms().